internal-packages/docs-scraper/scripts/utils/scrapeConfigs.ts

```diff
@@ -0,0 +1,118 @@
+	allNextConfigs: NextConfig[];
+	nextOnPagesDocumentedNextConfigs: string[];
+}>;
+export async function scrapeConfigs(
+	includeNextOnPages: false
+): Promise<{ allNextConfigs: NextConfig[] }>;
+export async function scrapeConfigs(includeNextOnPages = true): Promise<{
+	allNextConfigs: NextConfig[];
+	nextOnPagesDocumentedNextConfigs?: string[];
+}> {
+	const browser = await puppeteer.launch({
+		headless: 'new',
+	});
+	const page = await browser.newPage();
+
+	const allNextConfigs = await getNextConfigsFromNextDocs(page);
+
+	const nextOnPagesDocumentedNextConfigs = includeNextOnPages
+		? await getNextOnPagesDocumentedNextConfigs(page)
+		: undefined;
+
+	await browser.close();
+	return { allNextConfigs, nextOnPagesDocumentedNextConfigs };
+}
+
+async function getNextOnPagesDocumentedNextConfigs(
+	page: Page
+): Promise<string[]> {
+	await page.goto(
+		'https://github.com/cloudflare/next-on-pages/blob/main/packages/next-on-pages/docs/supported.md'
+	);
+	const nextOnPagesDocumentedNextConfigs = await page.$$eval(
+		'h3:has( > a[href="#nextconfigjs-properties"]) ~ table > tbody > tr > td:first-child',
+		els =>
+			els.map(el => {
+				// Note: footnotes leave a number at the end of the text, so we remove it here
+				return el.textContent.replace(/\d*$/, '');
+			})
+	);
+	return nextOnPagesDocumentedNextConfigs.filter(config => {
+		// for testing purposed let's pretend that haven't documented appDir and pageExtensions
+		return !['appDir', 'pageExtensions'].includes(config);
+	});
+}
+
+async function getNextConfigsFromNextDocs(page: Page): Promise<NextConfig[]> {
+	const pagesNextConfigsArray = await extractNextConfigsInfoFromNextDocsPage(
+		page,
+		'pages'
+	);
+	const appNextConfigsArray = await extractNextConfigsInfoFromNextDocsPage(
+		page,
+		'app'
+	);
+
+	const commonNextConfigs = appNextConfigsArray
+		.map(config => ({
+			app: config,
+			pages: pagesNextConfigsArray.find(
+				pagesConfig => pagesConfig.configName === config.configName
+			),
+		}))
+		.filter(configs => !!configs.pages)
+		.map(config => ({
+			configName: config.app.configName,
+			urls: [...config.app.urls, ...config.pages!.urls],
+		}));
+
+	const isCommonConfigName = (configName: string) =>
+		!!commonNextConfigs.find(
+			commonConfig => commonConfig.configName === configName
+		);
+
+	const appSpecificNextConfigs = appNextConfigsArray.filter(
+		config => !isCommonConfigName(config.configName)
+	);
+	const pagesSpecificNextConfigs = pagesNextConfigsArray.filter(
+		config => !isCommonConfigName(config.configName)
+	);
+
+	const configs = [
+		...commonNextConfigs,
+		...appSpecificNextConfigs,
+		...pagesSpecificNextConfigs,
+	].sort((a, b) => a.configName.localeCompare(b.configName));
+	return configs.filter(({ configName }) => {
+		// for testing purposed let's pretend that trailingSlash and urlImports have been removed from the next documentation
+		return !['trailingSlash', 'urlImports'].includes(configName);
+	});
```

Comment:
```suggestion
	return configs;
```
This is temporary just for testing :slightly_smiling_face: